packages/common/src/ui/components/NavLink.tsx

```diff
@@ -28,11 +29,24 @@ interface ListItemLinkProps {
   to: string;
 }
 
+const useSelectedNavMenuItem = (to: string, end: boolean): boolean => {
+  // This nav menu item should be selected when lower level elements
+  // are selected. For example, the route /customer-invoices/{id} should
+  // highlight the nav menu item for customer-invoices.
+  const highlightLowerLevels = !end || to.endsWith('*');
+
+  // If we need to highlight on lower levels, append a wildcard.
+  const path = highlightLowerLevels ? to : `${to}/*`;
```

Comment:
it works, though I'm confused by the wording. There's "If we need to highlight on lower levels, append a wildcard." but we append a wildcard if `highlightLowerLevels` is false

shouldn't the comment be more like 'If we need to highlight on higher levels'?

Reply:
Ugh I know!

Why this simple change was so hard for me 🙄 

All of the wording was difficult. I even tried a version where we just passed a `matchPath` in as a prop rather than doing this wildcard stuff here because I found it confusing and confusing to 'name'..